### ISSUE TITLE
Feature request: copy alt text 

### ISSUE THREAD

**Issue by colindean** [mastodon/mastodon-android#832]
I seem unable to (select and) copy the alt text for an image. 

A user went through great pains to transcribe a very long image text and like to use that instead of sharing the image, but I have no way to select the text or to push button copy it.

**Comment by grishka** [comment]
You can long tap on the alt text to select it:
![Screenshot_20240429-165330.png](https://github.com/mastodon/mastodon-android/assets/1478258/274ead7c-5ecc-4d6f-9c53-6c3a1f83ca8d)

**Comment by colindean** [comment]
Ah ha!

That works when the alt text is accessed from tapping on the `ALT` in the lower left corner of the image from the feed view or the single post view.

_However_, this was my workflow leading to me posting this:

When I am viewing the feed,

1. Tap on an image to view only the image
1. Tap on the ℹ️ icon in the upper right corner to show image actions and alt text

then I cannot interact with the alt text.

Tapping on the `ALT` is a sufficient workaround but I could see how users such as myself could get tripped up when viewing an image.

**Comment by grishka** [comment]
Oops I forgot about that one 🙃
